git-hooks/pre-commit

```diff
@@ -10,7 +10,13 @@ exec 1>&2
 
 changed_files=$(git diff --cached --name-only)
 changed_go_files=$(git diff --cached --name-only | grep -E '.go$' || true)
-all_go_files=$(find -name '*.go' | grep -v /vendor/)
+
+find_cmd="find"
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
```

Comment:
This implies that mac developer uses brew to install findutils which contains gfind.